js/web/app.js

```diff
@@ -28,23 +28,15 @@ const appInfo = {
             icon: "x-office-document"
           },
           routes: [
-            "files-personal",
-            "files-favorites",
-            "files-shared-with-others",
-            "files-shared-with-me",
-            "files-public-list"
+            "files-personal"
```

Comment:
Why did you remove all the route checks? The `routes` array defines the routes on which Collabora will be available. Should be working for `files-favorites`, `files-shared-with-others` and `files-shared-with-me` (at least for accepted shares). The public links page needs testing, not sure if it works there. If not you'd be right to remove it from the array of allowed routes.